lib/checkbufferoverrun.cpp

```diff
@@ -561,6 +561,9 @@ ValueFlow::Value CheckBufferOverrun::getBufferSize(const Token *bufTok) const
     if (!var)
         return ValueFlow::Value(-1);
 
+    if (var->isPointer())
```

Comment:
Could be merged with the previous condition.

Reply:
Done.

I initially had `dim` still before the condition so they were not next to each other.